pkg/utils/sysctl.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+
+	"github.com/golang/glog"
+)
+
+func SetSysctl(path string, value int) error {
+
+	sysctlPath := fmt.Sprintf("/proc/sys/%s", path)
+	if _, err := os.Stat(sysctlPath); err != nil {
+		if os.IsNotExist(err) {
+			glog.Infof("%s not found, could not set value: %s, Does your kernel support this feature?", sysctlPath)
+			return nil
+		}
+		glog.Errorf("error stating: %s : %s", sysctlPath, err.Error())
```

Comment:
same here, return err instead of log?

Reply:
some of these are non-fatal. f.x `conn_reuse_mode` where it does simply not exist in older kernels.

If i return a error this would exit nsc.Run

i'll add a third flag so we can say log or fail when calling SetSysctl for those values

Reply:
I was thinking more of having a new error type for non-fatal errors. The caller can check for that error and log or return err accordingly.

Reply:
how does one do a error-type for non-fatal errors? errors.New just gives me a error?